apps/homework/templates/homework/overview.html

```diff
@@ -15,6 +15,22 @@
         .no-padding {
```

Comment:
These css classes, `.no-padding` and `.condensed`, are defined but not used in multiple files including this one! Please address this.

Reply:
Removed from offending files. Placed in actual file where needed.